packages/styles/scss/components/modal/_modal.scss

```diff
@@ -108,6 +108,10 @@
     @include breakpoint(xlg) {
       width: 48%;
     }
+
+    .#{$prefix}--modal-container-body {
```

Comment:
@tay1orjones should the new styles be added to both `packages/styles/scss/components/modal/_modal.scss` and `packages/components/src/components/modal/_modal.scss`

Reply:
You may be able to get away with only `packages/components` in v10 at this point since the styles package is private, curious what you think @tay1orjones 👀  Definitely doesn't hurt having it in both places though (especially if that makes porting things back easier)

Reply:
Yeah, same thought. Having the same in both spots is preferred/nice but not necessary.